lib/ssh_scan/policy_manager.rb

```diff
@@ -104,6 +104,16 @@ def out_of_policy_auth_methods
       return outliers
     end
 
+    def minimum_ssh_version
+      target_ssh_version = @result[:ssh_version]
+      if @policy.ssh_version
+        if target_ssh_version < @policy.ssh_version
+          return @policy.ssh_version
+        end
+      end
+      return false
+    end
```

Comment:
Returning false or string is a bit weird here.

Maybe we need to improve terminology here.

Maybe there is a method that returns true/false, one that returns the version we saw, and one that returns the version we want?  I feel like that might be a little more deliberate.